pydicom/config.py

```diff
@@ -38,6 +46,36 @@ def reset_data_element_callback():
     data_element_callback_kwargs = {}
 
 
+def DS_numpy(use_numpy=True):
+    """Set whether multi-valued elements with VR of DS will be numpy arrays
+
+    .. versionadded:: 2.0
+
+    Parameters
+    ----------
+    use_numpy : bool, optional
+        ``True`` (default) to read multi-value DS elements
+           as `:class:~numpy.ndarray`
```

Comment:
```
:class:`~numpy.ndarray`
```
and 
```
:class:`~pydicom.multival.MultiValue`
```
Also the formatting here is a little weird and I'm not sure if its intention but maybe remove the indents?
below